pulpcore/pulpcore/app/viewsets/repository.py

```diff
@@ -239,11 +239,28 @@ class PublisherViewSet(CreateReadAsyncUpdateDestroyNamedModelViewset):
     filter_class = PublisherFilter
 
 
+class PublicationFilter(filterset.FilterSet):
```

Comment:
You don't need to add a filterset as part of this work. FilterSets are required for filtering over the REST API. The model layer queryset.filter() is unrelated.

Reply:
I guess we better have an explicit filter to only allow API filtering on the `Meta.fields`, without the `complete` field, so I'd rather keep this, probably changing the `parent.filter(complete=True)` to `parent.exclude(complete=True)`

Reply:
We can set the queryset to exclude(complete=False) on the Viewset.queryset instead, which will exclude for Read, List, and filter. As it is, v3/publications/ without filters will still include complete=False. I don't see a story to add filters to Publications but we should write one, and the filters we offer should be discussed. For instance "created" should probably be a date range filter.